src/engine/cachingreader/cachingreaderworker.cpp

```diff
@@ -221,6 +238,12 @@ void CachingReaderWorker::loadTrack(const TrackPointer& pTrack) {
             CachingReaderChunk::frames2samples(
                     m_pAudioSource->frameLength());
 
+    CuePointer pN60dBSound =
+            pTrack->findCueByType(mixxx::CueType::N60dBSound);
+    if (pN60dBSound) {
+        m_firstSoundFrameToVerify = pN60dBSound->getPosition();
+    }
+
```

Comment:
Can you add a comment that this is unneeded once `verifyFirstSound` has been removed (since we agreed that verifying the offsets in the cachingreaderworker is only a temporary workaround).

Reply:
Done.